templates/workspace_services/guacamole/user_resources/guacamole-azure-importreview-vm/delete_vm_extensions.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Actually I think you can delete this file, because it will get copied from the windowsvm template anyway